File: pe7.py
import time
def prime(n):
	f=0
	for i in range(1, n):
		f+=(n%i==0)
	return f==1
start=time.time()
n=1
k=1
while(k!=10001):
	n+=2
	if(prime(n)):
		k+=1
		#print(k, n)
print(n)
print(time.time()-start)

File: pe7.java
class pe7
{
	static boolean prime(int n)
	{
		for(int i=2;i<n;i++)
			if(n%i==0)
				return false;
		return true;
	}
	public static void main(String args[])
	{
		int i=1,n=1;
		while(n!=10001)
		{
			i+=2;
			if(prime(i)==true)
				n+=1;
		}
		System.out.print(i);
	}
}